piperider_cli/data/report/single-report/index.html

```diff
@@ -1 +1,28 @@
\ No newline at end of file
+<!doctype html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="initial-scale=1,width=device-width"/>
+  <link rel="apple-touch-icon" sizes="57x57" href="logo/apple-icon-57x57.png"/>
+  <link rel="apple-touch-icon" sizes="60x60" href="logo/apple-icon-60x60.png"/>
+  <link rel="apple-touch-icon" sizes="72x72" href="logo/apple-icon-72x72.png"/>
+  <link rel="apple-touch-icon" sizes="76x76" href="logo/apple-icon-76x76.png"/>
+  <link rel="apple-touch-icon" sizes="114x114" href="logo/apple-icon-114x114.png"/>
+  <link rel="apple-touch-icon" sizes="120x120" href="logo/apple-icon-120x120.png"/>
+  <link rel="apple-touch-icon" sizes="144x144" href="logo/apple-icon-144x144.png"/>
+  <link rel="apple-touch-icon" sizes="152x152" href="logo/apple-icon-152x152.png"/>
+  <link rel="apple-touch-icon" sizes="180x180" href="logo/apple-icon-180x180.png"/>
+  <link rel="icon" type="image/png" sizes="192x192" href="logo/android-icon-192x192.png"/>
+  <link rel="icon" type="image/png" sizes="32x32" href="logo/favicon-32x32.png"/>
+  <link rel="icon" type="image/png" sizes="96x96" href="logo/favicon-96x96.png"/>
+  <link rel="icon" type="image/png" sizes="16x16" href="logo/favicon-16x16.png"/>
+  <link rel="manifest" href="logo/manifest.json"/>
+  <meta name="msapplication-TileColor" content="#ffffff"/>
+  <meta name="msapplication-TileImage" content="/ms-icon-144x144.png"/>
+  <meta name="theme-color" content="#ffffff"/>
+  <script defer="defer" src="./static/js/main.fdc84d1b.js"></script>
+</head>
+<body>
+<div id="root"></div>
+<script>window.PIPERIDER_SINGLE_REPORT_DATA = "", window.PIPERIDER_COMPARE_REPORT_DATA = ""</script>
```

Comment:
I understood that these vars will be replaced, but I still think the var `PIPERIDER_COMPARE_REPORT_DATA` should be renamed to `PIPERIDER_COMPARISON_REPORT_DATA`

Reply:
good point. I'll change these.